ui/app/src/modules/Preview/Tools/AudiencesPanel.tsx

```diff
@@ -120,18 +125,28 @@ export function AudiencesPanelUI(props: AudiencesPanelUIProps) {
         <>
           <Grid className={classes.PanelMargin}>
             {
-              Object.keys(contentType.fields).map((field: any) => (
-                <React.Fragment key={field}>
-                  <AudiencesFormSection
-                    property={contentType.fields[field]}
-                    modelApplying={modelApplying}
-                    modelValue={model[field] ? model[field].key : undefined}
-                    modelTimezone={model[`${field}_tz`] ? model[`${field}_tz`].key : undefined}
-                    onFormChange={onFormChange}
-                  />
-                  <Divider className={classes.divider}/>
-                </React.Fragment>
-              ))
+              Object.keys(contentType.fields).map((field: any) => {
+                const type = contentType.fields[field].type;
+                const Control = controlsMap[type];
+
+                if (model[`${field}_tz`]) {
+                  contentType.fields[field].timezone = model[`${field}_tz`];
+                }
+
+                return (
+                  <React.Fragment key={field}>
+                    <AudiencesFormSection field={contentType.fields[field]}>
+                      <Control
+                        field={contentType.fields[field]}
+                        value={model[field] ? model[field].key : undefined}
+                        onChange={onFormChange}
+                        disabled={modelApplying}
+                      />
+                    </AudiencesFormSection>
+                    <Divider className={classes.divider}/>
+                  </React.Fragment>
```

Comment:
1. move key to FormSection
2. move divider to FormSection
3. remove Fragment
4. add `showDivider: boolean` prop to FormSection